heapster-saw/examples/_CoqProject

```diff
@@ -4,7 +4,7 @@
 
 # FIXME: Uncomment _proofs files when they're updated with the latest automation
 linked_list_gen.v
-linked_list_proofs.v
+#linked_list_proofs.v
```

Comment:
Do you plan to re-enable `linked_list_proofs.v` (or one of the other tests removed below) at some point? If so, it would be worth tracking this in an issue, along with a description of what remains to be done (i.e., why doesn't it work with the new version of MRSolver?)

Reply:
Yes, that's a good point, I'll file an issue